### PR TITLE
Changes img HTML tag to  Image Component and NavBar Component to the pages.

### PR DESCRIPTION
**Pull request by PremanandChowdhury**
Issue No. : #114 
Please Check the PR and mention the changes/improvements, if any.
Thank You.

### CODE REVIEW

File: components/NavBar/index.js [hunk 1]
```diff
@@ -8,6 +9,18 @@ const NavBar = ({ personData: { photo } }) => {
     <nav className={styles.navBar}>
       <img className={styles.logo} src={RDSLogo} alt="RealDevSquad Logo" />
```
Comment (Kratika0907) [inline components/NavBar/index.js:10]: why the logo is there two times?

File: components/NavBar/index.js [hunk 2]
```diff
@@ -8,6 +9,18 @@ const NavBar = ({ personData: { photo } }) => {
...
+            src={RDSLogo}
+            alt="RealDevSwaud Logo"
+            width="111"
+            height="111"
+            layout="fixed"
+          />
+        </div>
+      </li>
+
+      {/* <li>
```
Comment (Kratika0907) [inline components/NavBar/index.js:23]: Can you put these link back in nav bar. On left side we can have them.

File: components/filter/index.js [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
-        alt="Filter icon"
-        onClick={() => {
-          document
-            .getElementById('filterContent')
-            .classList.toggle(styles.showList);
-        }}
-      />
+      <div className="icon">
+        <Image
+          src="/assets/filter-icon.svg"
```
Comment (Kratika0907) [inline components/filter/index.js:10]: Somewhere it is path.join and some where direct string. can we have a consistent style for same.

File: components/product-details/index.js [hunk 4]
```diff
@@ -1,24 +1,33 @@
...
+            height={200}
+          />
+        </div>
+
+        <div className="product-details">
+          <h2> {productJSON.name}</h2>
+          <h3> Where to use</h3>
+          <ul>
+            {productJSON.usage.map((use, index) => {
+              return <li key={index}>{use}</li>;
```
Comment (sumitd94) [inline components/product-details/index.js:26]: Using `index` as the key is not a good practise. Are we not getting any product id from the `productJSON` ?

File: pages/shop/index.js [hunk 5]
```diff
@@ -11,59 +11,64 @@ import {
...
+          <div className="take-to-cart">
+            <button>
+              <Link href="/cart">Take me to cart</Link>
+            </button>
+          </div>
+          <div className="shoppinglist-container">
+            {products.map((itemName) => {
+              return (
+                <ShopCard
+                  key={itemName}
```
Comment (sumitd94) [inline pages/shop/index.js:36]: If the `products` array does not contain any unique id, try using this package [uuid](https://www.npmjs.com/package/uuid)

File: components/filter/index.js (hunk 3 above)
Comment (PremanandChowdhury) [reply components/filter/index.js:10]: yes.

**Pull request closed without merging**